agent/tcs/handler/handler.go

```diff
@@ -84,7 +84,7 @@ func StartSession(params *TelemetrySessionParams, statsEngine stats.Engine) erro
 			seelog.Info("TCS Websocket connection closed for a valid reason")
 			backoff.Reset()
 		} else {
-			seelog.Infof("Error from tcs; backing off: %v", tcsError)
+			seelog.Errorf("Error: lost websocket connection with ECS Telemetry service (TCS): %v", tcsError)
```

Comment:
Can this be warn instead of errorf, customers look at these logs and think that something is wrong with the Agent when essentially it is not sending metrics to tcs.

Reply:
I prefer it to be an error because the agent should have a connection to TCS at all times. In my experience losing connection with TCS often indicates a deeper issue on the instance and indicates that an error has occurred.